Makefile

```diff
@@ -103,6 +103,12 @@ Q := $(if $V,,@)
 	$Q ln -s ../bin .GOPATH/bin
 	$Q touch $@
 
+.GOPATH/.setup:
```

Comment:
Why not add it to .GOPATH/.ok? I really don't want to expose another internal to the user.